src/platform/windows/display.h

```diff
@@ -338,50 +344,285 @@ namespace platf::dxgi {
   };
 
   /**
-   * Display duplicator that uses the Windows.Graphics.Capture API.
+   * Display backend that uses Windows.Graphics.Capture with a software encoder.
+   * This now always uses IPC implementation via display_wgc_ipc_ram_t.
    */
-  class wgc_capture_t {
-    winrt::Windows::Graphics::DirectX::Direct3D11::IDirect3DDevice uwp_device {nullptr};
-    winrt::Windows::Graphics::Capture::GraphicsCaptureItem item {nullptr};
-    winrt::Windows::Graphics::Capture::Direct3D11CaptureFramePool frame_pool {nullptr};
-    winrt::Windows::Graphics::Capture::GraphicsCaptureSession capture_session {nullptr};
-    winrt::Windows::Graphics::Capture::Direct3D11CaptureFrame produced_frame {nullptr}, consumed_frame {nullptr};
-    SRWLOCK frame_lock = SRWLOCK_INIT;
-    CONDITION_VARIABLE frame_present_cv;
-
-    void on_frame_arrived(winrt::Windows::Graphics::Capture::Direct3D11CaptureFramePool const &sender, winrt::Windows::Foundation::IInspectable const &);
-
+  class display_wgc_ram_t {
   public:
-    wgc_capture_t();
-    ~wgc_capture_t();
+    /**
+     * @brief Factory method for initializing WGC RAM capture backend.
+     *
+     * Always returns the IPC implementation for Windows.Graphics.Capture using a software encoder.
+     *
+     * @param config Video configuration parameters.
```

Comment:
```suggestion
    /**
     * @brief Factory method for initializing WGC RAM capture backend.
     * Always returns the IPC implementation for Windows.Graphics.Capture using a software encoder.
     * @param config Video configuration parameters.
```

Please remove or reduce empty lines in documentation blocks.